tests/php/test-amp-scribd-embed-handler.php

```diff
@@ -21,22 +27,10 @@ class AMP_Scribd_Embed_Handler_Test extends WP_UnitTestCase {
 
 	/**
 	 * Set up.
-	 *
-	 * @global WP_Post $post
 	 */
 	public function setUp() {
-		global $post;
 		parent::setUp();
-
-		/*
-		 * As #34115 in 4.9 a post is not needed for context to run oEmbeds. Prior to 4.9, the WP_Embed::shortcode()
-		 * method would short-circuit when this is the case:
-		 * https://github.com/WordPress/wordpress-develop/blob/4.8.4/src/wp-includes/class-wp-embed.php#L192-L193
-		 * So on WP<4.9 we set a post global to ensure oEmbeds get processed.
-		 */
-		if ( version_compare( strtok( get_bloginfo( 'version' ), '-' ), '4.9', '<' ) ) {
-			$post = self::factory()->post->create_and_get();
-		}
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/4901/commits/f9d36eed14119a2a3a7b9b59e8ec1cf3a41c4618.